alt-tab-macos/logic/PreferencesPanelNSTextFieldEditable.swift

```diff
@@ -0,0 +1,34 @@
+import Cocoa
+
+class PreferencesPanelNSTextFieldEditable: NSTextField, NSTextFieldDelegate {
```

Comment:
Oh, yeah sorry for that.

The naming is hard one this one. Thats why I went with naming this in relation to the PreferencesPanel. In generic wording this would be something like ValidatedInstantChangeNSTextField?

Reply:
If there is only 1 type of TextField in the app, I think we can just call it `TextField`, no? I think it's not very ambiguous, and you can just see the call-sites if the context is unclear

Reply:
Sure. Btw. had to add an second NSTextField but named them ...Label as you proposed and this one just TextField.